ansible/tasks/setup-pgbouncer.yml

```diff
@@ -77,6 +77,12 @@
     - 'generated-optimizations.ini'
     - 'custom-overrides.ini'
 
+- name: PgBouncer - custom-overrides.ini
+  copy:
+    dest: /etc/pgbouncer-custom/custom-overrides.ini
+    content: |
+      listen_addr = 0.0.0.0
```

Comment:
just edit the config we copy over - no reason this needs to be a custom override